setup.py

```diff
@@ -8,7 +8,8 @@
     name="phcode42v2",
     version="1.0.0",
     description="Code42 for Phantom",
-    packages=find_packages(),
+    packages=find_packages("src"),
```

Comment:
This is what ultimately should fix the issue with some IDEs not being able to debug through tests. Basically, without this, `pip install` causes the tests to get installed too, which causes weirdness.